concepts/05 Widgets/Scheduler/077 Time Zone Support.md

```diff
@@ -1,21 +1,11 @@
-Appointments are displayed in the client time zone. The time-zone offset specified in the [startDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/startDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDate') or [endDate](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/endDate.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDate') is also taken into account (if there is any). You can force a time zone for an individual appointment via its [startDateTimeZone](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/startDateTimeZone.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#startDateTimeZone') and [endDateTimeZone](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/endDateTimeZone.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#endDateTimeZone') fields or for all appointments in the widget via its [timeZone](/api-reference/10%20UI%20Widgets/dxScheduler/1%20Configuration/timeZone.md '/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone') option. **timeZone** takes precedence over **start**/**endDateTimeZone**.
+The **Scheduler** displays appointments according to the client machine's time zone. If you want to override client settings, use the [timeZone](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone) option.
```

Comment:
```suggestion
The **Scheduler** displays appointments according to the client time zone. If you want to override client settings, use the [timeZone](/Documentation/ApiReference/UI_Widgets/dxScheduler/Configuration/#timeZone) option.
```
Scheduler can be used on a portable device that's why 'machine' should better be omitted.